data/human/free worlds 3 reconciliation.txt

```diff
@@ -958,7 +958,7 @@ mission "FW Syndicate Capture 1C"
 mission "FW Pug 1"
 	landing
 	name "What happened to Rand?"
-	description "Try to determine why communications with Rand in the Zeta Aquilae system were cut off. If you cannot reach Rand, at least look for anything odd in the neighboring systems."
+	description "Try to determine why communications with Rand in the Zeta Aquilae system were cut off. Look for clues in neighboring systems."
```

Comment:
```suggestion
	description "Try to determine why communications with Rand in the Zeta Aquilae system were cut off."
```
Probably shouldn't even mention the neighboring system at all.

Reply:
Well you need to go through the neighboring systems to try to get to Rand in the first place. How could the player possibly miss them?